src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/PackageSearchMetadataCacheItemEntry.cs

```diff
@@ -4,6 +4,8 @@
 #nullable enable
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Sort/remove usings

Reply:
Sort/remove should not cause additional using statements. 
If there's no reason for that, revert this file.